source/addComment.js

```diff
@@ -1,3 +1,54 @@
 $(document).ready(function(){
-  // add your code here
+  $('form').on('submit', function(event) {
+    event.preventDefault()
+  })
+  var newCommentButton = document.getElementById('new_comment_button')
+  var submitButton = document.getElementsByTagName('input')[1]
+
+  newCommentButton.addEventListener('click', renderCommentForm)
+  submitButton.addEventListener('click', postComment)
 })
+
+function renderCommentForm() {
+  var commentForm = document.getElementById('new_comment')
```

Comment:
sometimes you are using javascript and sometimes jquery to grab elements in the DOM. I would stick to just 1. I would recommend jQuery to get that practice and its a bit easier